Assets/Scripts/Layers/LayerTypes/LayerNL3DBase.cs

```diff
@@ -91,31 +76,39 @@ public bool ActiveInHierarchy
             }
         }
 
-        public readonly UnityEvent<string> NameChanged = new();
-        public readonly UnityEvent<bool> LayerActiveInHierarchyChanged = new();
-        public readonly UnityEvent<Color> ColorChanged = new();
-        public readonly UnityEvent LayerDestroyed = new();
+        [JsonIgnore] public readonly UnityEvent<string> NameChanged = new();
```

Comment:
Aren't UnityEvents always ignored because they are not serializable?

Reply:
no, they will show up in the JSON

Reply:
Really? 😞